sematic/storage.py

```diff
@@ -79,6 +113,17 @@ def get(self, key: str) -> bytes:
         except FileNotFoundError:
             raise NoSuchStorageKey(self, key)
 
+    def _get_write_location(self, namespace: str, key: str) -> str:
+        return os.path.join(get_config().data_dir, namespace, key)
+
+    def _get_read_location(self, namespace: str, key: str) -> str:
+        return f"sematic:///data/{namespace}/{key}"
+
+
+class S3ClientMethod(enum.Enum):
+    PUT = "put_object"
+    GET = "get_object"
+
 
 class S3Storage(Storage):
```

Comment:
Please review this Server Settings Split [discussion thread](https://github.com/sematic-ai/sematic/pull/330#discussion_r1024659434).

`AWS_S3_BUCKET` should be exclusively a server-side settings, and the S3Storage location should come bundled with the bucket information. So instead of having `S3Storage` extract the bucket from the user settings, it should get it from the API call.

Reply:
@neutralino1 We discussed offline that I would temporarily move `AWS_S3_BUCKET` back to a user setting, and that in this PR you would need to move again to be a server setting.

All you need to do is revert [this commit](https://github.com/sematic-ai/sematic/pull/330/commits/28f7a7db34212b41cdce20758306a82d019d9c22), and update [this documentation entry](https://github.com/sematic-ai/sematic/pull/330/commits/28f7a7db34212b41cdce20758306a82d019d9c22#diff-27c8c22b45605a0ddf42ce8017b6b2a68cb3b10534c18bf4a781e230d7b92361R95-R96) to reflect your changes:
```
- `AWS_S3_BUCKET`: the S3 bucket to use for persisting artifacts; required only
  for cloud pipeline submissions
```